torch/quantization/_quantize_script.py

```diff
@@ -81,8 +81,7 @@ def convert_script(model, inplace=False):
     if not inplace:
         model = model.copy()
     model = wrap_cpp_module(torch._C._jit_pass_insert_quant_dequant(model._c, 'forward', False))
-    if 'fbgemm' in torch.backends.quantized.supported_engines:
-        torch._C._jit_pass_insert_prepack_unpack(model._c)
+    model = wrap_cpp_module(torch._C._jit_pass_quant_finalize(model._c))
```

Comment:
Please add an option to convert_script to enable/disable finalize so that users have the ability to dump the model prior to finalization. To make it simple for users, suggest
```python
convert_script(model, inplace=False, debug=False):
 
  if not debug:
       model = wrap_cpp_module(torch._C._jit_pass_quant_finalize(model._c))
```

Reply:
It is important to add this option to both quantize and convert script so that users can examine a model with quant/dequant ops prior to freezing and inlining.